siliconcompiler/tools/surelog/surelog.py

```diff
@@ -133,10 +135,20 @@ def post_process(chip):
     '''
     design = chip.get('design')
     step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    tool = 'surelog'
 
     if step != 'import':
         return 0
 
+    # Log file parsing
+    chip.set('eda', tool, 'regex', step, index, 'warnings', "WARNING")
+    chip.set('eda', tool, 'regex', step, index, 'errors', "ERROR")
+
+    # Output reprts for deep dive
+    for metric in ('errors', 'warnings'):
+        chip.set('eda', tool, 'report', step, index, metric, f"{step}.log")
```

Comment:
I think these paths cannot be found in the `check_manifest` test because they are relative to the step directory, not the top-level build directory.

```suggestion
    # Output reprts for deep dive
    logfile = os.path.join(os.getcwd(), f"{step}.log")
    local_logfile = logfile[logfile.rfind(chip.get('dir')):]
    for metric in ('errors', 'warnings'):
        chip.set('eda', tool, 'report', step, index, metric, local_logfile)
```

Reply:
Nice debugging, and good suggestion. One other thing we could consider is handling these report keypaths as a special case in `chip.find_files()`, which I generally wouldn't be a fan of except there is precedent for doing it for the 'input'/'output' keypaths (which are very similar): https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/core.py#L1279-L1290.